### DISCUSSION TITLE
Export all application assignments as a single CSV

### DISCUSSION THREAD

**Discussion by BARmichael-fleet** [Micke-K/IntuneManagement#179] [General]
Hi. Is it possible to export a list of all Intune application assignments within a single CSV file? For example -

Application Name, Required Assignments, Available Assignments, Uninstall Assignments

**Comment by Micke-K** [comment]
Hello

There is no good way of doing this at the moment. I have a few ideas and might add this or something similar in a future version.

Cheers!

**Comment by MariuszFerdyn** [comment]
The same for all other resources like DeviceConfiguration, UpdatePolicies - It will be very usefull for reports. From other way the powershell script should do it also, get what we have currently and put it to sigle CSV.
